style.css

```diff
@@ -0,0 +1,334 @@
+@media (max-width: 319px) {
+
+}
+
+@media (min-width: 320px) and (max-width: 374px) {
+
+}
+
+@media (min-width: 375px) and (max-width: 767px) {
+
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+	.wrapper {
+		width: 768px;
+	}
+	.header {
+		grid-template-columns: 1fr 1fr;
+	}
+	.header__menu, .header__search, .header__user-block, .header__cart, .header__language-block {
+		display: none;
+	}
+	.menu-bar {
+		display: flex;
+		justify-content: flex-end;
+		align-items: center;
+	}
+	.top-content__h1 {
+		font-family: 'GothamPro-bold', serif;
+		font-size: 50px;
+		line-height: 48px;
+	}
+	.link {
+		font-size: 20px;
+		line-height: 19px;
+	}
+	.top-content {
+		height: 415px;
+	}
+}
+
+* {
+	padding: 0;
+	margin: 0;
+}
+
+.wrapper {
+	max-width: 930px;
+	min-width: 320px;
+	font-family: 'GothamPro', serif;
+	margin: 0 auto;
+	box-sizing: border-box;
+}
+
+/*header*/
+
+.header {
+	position: relative;
+	width: 100%;
+	height: 96px;
+	display: grid;
+	flex-direction: row;
+	grid-template-columns: 0.7fr 2.7fr 1fr 1.5fr 0.5fr 0.7fr 0.3fr;
+	margin-bottom: 34px;
+}
+
+.menu-bar {
+	display: none;
+}
+
+.header__logo {
+	display: flex;
+	align-items: center;
+	justify-content: flex-start;
+}
+
+.header__logo-image {
+	width: 55px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)